### PR TITLE
Alerting: Expose metrics for Alertmanager Alerts - `grafana_alerting_alertmanager_alerts`

### PR DESCRIPTION
**Pull request by gotjosh**
In Grafana, the alert evaluation and alert delivery are combined. We've always used a metric named `grafana_alerting_alerts` to get a sense of what are the alerts that are currently firing (these come from the evaluation side) and opted to not map the alertmanager alerts metric directly.

I think it's important that we make a distinction between alerts that happen at evaluation vs alerts that are received for delivery by the internal Alertmanager as we have options to skip the delivery of these alerts to the internal alertmanager altogether.

### CODE REVIEW

File: pkg/services/ngalert/state/manager.go [hunk 1]
```diff
@@ -19,8 +19,7 @@ import (
 )
 
 var (
-	ResendDelay           = 30 * time.Second
-	MetricsScrapeInterval = 15 * time.Second // TODO: parameterize? // Setting to a reasonable default scrape interval for Prometheus.
```
Comment (gotjosh) [inline pkg/services/ngalert/state/manager.go:23]: drive-by fix as this is no longer used.

**Review by alexweav — APPROVED** [review]
